service/replication.go

```diff
@@ -278,6 +278,18 @@ func (s *service) CreateStorageProtectionGroup(ctx context.Context,
 	}, nil
 }
 
+// TODO: implement
+// DeleteLocalVolume deletes the backend volume on the storage array.
+func (s *service) DeleteLocalVolume(ctx context.Context,
+	req *csiext.DeleteLocalVolumeRequest) (*csiext.DeleteLocalVolumeResponse, error) {
+	ctx, log, _ := GetRunIDLog(ctx)
+
+	log.Info("!!! Deleting Remote Volume !!!")
+	log.Error("DeleteLocalVolume is not yet implemented")
```

Comment:
DeleteLocalVolume on PowerScale will be a NOP. We could probably print that the local backend folder will be deleted upon the next sync and call it a day, here. :)